.travis.yml

```diff
@@ -1,22 +1,26 @@
 language: php
 
 php:
-  - 5.3
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
+  - hhvm
+
+matrix:
+  allow_failures:
+    - php: hhvm
+    - php: 7.0
 
 services:
-  - apc
   - memcache
-  - memcached # will start memcached
+  - memcached
 
 before_script:
   - composer update
```

Comment:
As I learnd from Rafael Dohms, `composer install` should be used to install the dependencies as recorded in `composer.lock`.